caveclient/chunkedgraph.py

```diff
@@ -485,7 +485,7 @@ def get_operation_details(self, operation_ids: Iterable[int]):
         return r.json()
 
     def get_lineage_graph(
-        self, root_id, timestamp_past=None, timestamp_future=None, as_nx_graph=False
+        self, root_id=None, root_ids=[], timestamp_past=None, timestamp_future=None, as_nx_graph=False
```

Comment:
`root_id` is ignored by the function but is needed for supporting existing implementations using this function. Please implement this in a way that both an int and a list is supported to be passed to `root_id` and remove `root_ids` from the parameter list.